src/main/java/me/jellysquid/mods/lithium/common/world/NullBlockEntityList.java

```diff
@@ -0,0 +1,128 @@
+package me.jellysquid.mods.lithium.common.world;
+
+import net.minecraft.block.entity.BlockEntity;
+import net.minecraft.world.World;
+
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ListIterator;
+
+public class NullBlockEntityList<E extends BlockEntity> implements List<E> {
+    @Override
+    public int size() {
+        throw new UnsupportedOperationException("Lithium does not support this");
+    }
+
+    @Override
+    public boolean isEmpty() {
+        return false;
+    }
+
+    @Override
+    public boolean contains(Object o) {
+        return false;
+    }
+
+    @Override
+    public Iterator<E> iterator() {
+        throw new UnsupportedOperationException("Lithium does not support this");
```

Comment:
If this was taken from turtlemod, I made a PR to it before you archived it fixing an error this causes when shutting down the server, make a singleton empty iterator or iirc java / guava provides one